src/main/java/org/dcsa/core/security/SecurityConfig.java

```diff
@@ -53,13 +53,15 @@ public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
                     .oauth2ResourceServer()
                     .jwt()
                     .jwtAuthenticationConverter(new JwtAuthenticationConverter())
-                    .and();
+                    .and()
```

Comment:
Two rows with same content?!?

Reply:
Looks a bit weird, but is is needed. The first and() returns a OAuth2ResourceServerSpec object, and the second one (used on the OAuth2ResourceServerSpec) returns a ServerHttpSecurity Object, where cors can be enabled